site/content/docs/0.0/utilities/spacing.md

```diff
@@ -1,11 +1,362 @@
+- `b` - for classes that set `margin-bottom` or `padding-bottom`
+- `s` - (start) for classes that set `margin-left` or `padding-left` in LTR, `margin-right` or `padding-right` in RTL
+- `e` - (end) for classes that set `margin-right` or `padding-right` in LTR, `margin-left` or `padding-left` in RTL
+- `x` - for classes that set both `*-left` and `*-right`
+- `y` - for classes that set both `*-top` and `*-bottom`
+- blank - for classes that set a `margin` or `padding` on all 4 sides of the element
+
+Where *size* is one of:
+
+- `0` - for classes that eliminate the `margin` or `padding` by setting it to `0`
+- `1` - (by default) for classes that set the `margin` or `padding` to `$spacer * .25`
+- `2` - (by default) for classes that set the `margin` or `padding` to `$spacer * .5`
+- `3` - (by default) for classes that set the `margin` or `padding` to `$spacer`
+- `4` - (by default) for classes that set the `margin` or `padding` to `$spacer * 1.5`
+- `5` - (by default) for classes that set the `margin` or `padding` to `$spacer * 3`
+- `auto` - for classes that set the `margin` to auto
+
+(You can add more sizes by adding entries to the `$spacers` Sass map variable.)
+
+{{< /bootstrap-compatibility >}}
+
+### Notation for scaled values
+
+Spacing utilities that evolve across all breakpoints, from `2xs` to `3xl`, have no breakpoint abbreviation in them since they are auto-sufficient from a responsive point of view.
+
+The classes are named using the format `{property}{sides}-scaled-{size}`.
+
+Where *property* is one of:
+
+- `m` - for classes that set `margin`
+- `p` - for classes that set `padding`
+
+Where *sides* is one of:
+
+- `t` - for classes that set `margin-top` or `padding-top`
+- `b` - for classes that set `margin-bottom` or `padding-bottom`
+- `s` - (start) for classes that set `margin-left` or `padding-left` in LTR, `margin-right` or `padding-right` in RTL
+- `e` - (end) for classes that set `margin-right` or `padding-right` in LTR, `margin-left` or `padding-left` in RTL
+- `x` - for classes that set both `*-left` and `*-right`
+- `y` - for classes that set both `*-top` and `*-bottom`
+- blank - for classes that set a `margin` or `padding` on all 4 sides of the element
+
+Where *size* is one of:
+
+{{< bs-table "table" >}}
+| Size name | Size for mobile (`2xs` to `md`) | Size for tablet (`md` to `xl`) | Size for desktop (`xl` to `3xl` and above) |
+| --- | --- | --- | --- |
+| `none` | `0` | `0` | `0` |
+| `smash` | `2px` | `4px` | `4px` |
+| `shortest` | `4px` | `8px` | `8px` |
+| `shorter` | `8px` | `12px` | `16px` |
+| `short` | `12px` | `16px` | `24px` |
+| `medium` | `16px` | `24px` | `32px` |
+| `tall` | `24px` | `32px` | `40px` |
+| `taller` | `32px` | `40px` | `48px` |
+| `tallest` | `40px` | `48px` | `56px` |
+| `spacious` | `48px` | `56px` | `64px` |
+{{< /bs-table >}}
+
+(You can add more sizes by adding entries to the `$ouds-dimension-space-scaled` Sass map variable.)
+
+### Examples
```

Comment:
I don't know if the scaled values utilities are clear enough for users. Maybe we could have a "live" example like the one for horizontal centering, so the user can see the margin or padding changing while he resize is screen ?